lua/examples/policy.lua

```diff
@@ -171,7 +171,7 @@ local bogons_ipv4_file = "lua/examples/bogons-ipv4.txt"
 local num_ipv4_rules, num_ipv4_tbl8s = lpm_para_estimate(bogons_ipv4_file)
 num_ipv4_rules = math.max(1, scaling_factor_rules * num_ipv4_rules)
 num_ipv4_tbl8s = math.max(1, scaling_factor_tbl8s * num_ipv4_tbl8s)
-local lpm = lpmlib.new_lpm(num_ipv4_rules, num_ipv4_tbl8s)
+lpm = lpmlib.new_lpm(num_ipv4_rules, num_ipv4_tbl8s)
```

Comment:
Add a comment to explain why we made `lpm` global and city file `lua/examples/example_gt_lpm_params_request.lua`.